client/components/_constants.tsx

```diff
@@ -1,2 +1,2 @@
-export const SERVER_URL: string = 'https://beefreecommunity.org';
-// export const SERVER_URL: string = 'http://localhost:3001';
+//export const SERVER_URL: string = 'https://beefreecommunity.org';
+ export const SERVER_URL: string = 'http://localhost:3001';
```

Comment:
This should really be handled by environment variables, but for now we need to make sure we don't commit setting it to "localhost"